laravel/tests/Unit/EvaluationParserTest.php

```diff
@@ -17,9 +15,136 @@ class EvaluationParserTest extends TestCase
      */
     public function testEvaluationParser()
     {
-        $parser = new EvaluationParser("20180418", true);
-        $this->assertDatabaseHas("evaluation_data", [
-            "date" => date("2018-04-17"),
+        $parser = new EvaluationParser('20210323', true);
```

Comment:
This will be removed later - is on test but will just send me emails, not David so don't worry about running tests

Reply:
I plan on giving it a file to parse and checking if the data is in the table. I'm going to use refreshDatabase and hopefully don't run into any issues.

Reply:
Should be added now.